plugins/AudioFileProcessor/AudioFileProcessorWaveView.cpp

```diff
@@ -395,7 +403,7 @@ void AudioFileProcessorWaveView::slideSamplePointByPx(Point point, int px)
 	);
 }
 
-void AudioFileProcessorWaveView::slideSamplePointByFrames(Point point, f_cnt_t frames, bool slide_to)
+void AudioFileProcessorWaveView::slideSamplePointByFrames(Point point, long frames, bool slide_to)
```

Comment:
If this is a loop variable, better to use `size_t` and make the change in loops too accordingly

Reply:
The problem was that `frames` could be negative, so I changed it to long. I can change it to `ssize_t` if you want.

Reply:
Nope this is better then

Reply:
Good catch @szeli1, it could be negative, so it could be a signed integer variable rather than unsigned. These were recently changed to `size_t`, so I was questioning why these were changed, but I realized that these can infact be negative.